src/v/iceberg/tests/BUILD

```diff
@@ -351,3 +351,28 @@ redpanda_cc_gtest(
         "@googletest//:gtest",
     ],
 )
+
+redpanda_cc_gtest(
+    name = "schema_protobuf_test",
+    timeout = "short",
+    srcs = [
+        "schema_protobuf_test.cc",
+    ],
+    data = [
+        "testdata/iceberg_ready_test_messages_edition2023.proto",
```

Comment:
If you want to make this work in CMake (I am fine if you don't you can make it an environment variable and do the same in CMake).

Env usage in Bazel is something like:

```
env = {
  "CONFORMANCE_PROTO": "$(rootpath testdata/iceberg_ready_test_messages_edition2023.proto)"
}
```

Reply:
it actually works well in CMake thanks to Andrews: `test_utils::get_runfile_path`

Reply:
Our runfile code is a gift from @dotnwat 😎